vispy/glsl/build-spatial-filters.py

```diff
@@ -654,9 +655,37 @@ def weight(self, x):
 print("")
 print("const float kernel_bias  = %f;" % bias)
 print("const float kernel_scale = %f;" % scale)
+print("const float kernel_size = %f;" % n)
 print("uniform sampler2D u_kernel;")
 print("")
 
+code = 'float\n'
+code += 'unpack(vec4 rgba)\n'
```

Comment:
let's call this `unpack_unit` (and same thing for the `pack` functiona above) because it only packs/unpacks things in the unit range, not any float